Dockerfile

```diff
@@ -43,7 +38,7 @@ RUN cp ./ocm-backplane /out
 RUN chmod -R +x /out
 
 ### Build dependencies
-FROM registry.access.redhat.com/ubi9/ubi:9.5 as dep-builder
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.23 as dep-builder
```

Comment:
While I do agree on the unavailability of yum in rhel_9, I wonder is it a sustainable solution to move back to rhel_8 because of that?

Reply:
Thanks @MitaliBhalla , It works